versions.mk

```diff
@@ -0,0 +1,34 @@
+
+FIO_VER ?= 3.15
+TELEPORT_TAG ?= 3.2.17
+ETCD_VER ?= 2.3.7
+# abbreviated gravity version to use as a build ID
+GRAVITY_VERSION ?= $(shell ./version.sh)
+# current Kubernetes version
+K8S_VER := 1.19.12
+# Kubernetes version suffix for the planet package, constructed by concatenating
+# major + minor padded to 2 chars with 0 + patch also padded to 2 chars, e.g.
+# 1.13.5 -> 11305, 1.13.12 -> 11312, 2.0.0 -> 20000 and so on
+K8S_VER_SUFFIX ?= $(shell printf "%d%02d%02d" $(shell echo $(K8S_VER) | sed "s/\./ /g"))
+PLANET_TAG ?= 8.0.3-$(K8S_VER_SUFFIX)
+PLANET_BRANCH ?= $(PLANET_TAG)
```

Comment:
Do we need `PLANET_BRANCH`? I removed it in https://github.com/gravitational/gravity/pull/2611 since it seemed like it was used interchangeably with `PLANET_TAG` and was partially removed in a prior PR.

Reply:
Ok - I don't think there's inherent necessity in the branch variable - thanks for the heads up on your PR.